packages/server/src/config.ts

```diff
@@ -112,6 +112,7 @@ export interface MedplumDatabaseConfig {
   password?: string;
   ssl?: MedplumDatabaseSslConfig;
   queryTimeout?: number;
+  maxConnectionsPerServer?: number;
```

Comment:
This might be artifact of old PR?  This interface already contains `maxConnections`